pedsim_relay:
  pedsim_tracker:
    ros_parameters:
      # Subscriber topics
      odom_topic: "locobot/odom"
      detected_agents_topic: "human/simulated_agents"
      # Publisher topics
      human_track_topic: "human/interpolated_history"
      costmap_topic: "global_costmap/costmap"
      pub_frame_id: "locobot/odom"
      # Parameters
      interp_interval: 0.4      # time interval between interpolation points [s]
      max_history_length: 7     # maximum number of history points (without current)
      max_num_agents: 5         # maximum number of agents to be considered for motion planning
      track_timeout: 0.15       # maximum time to keep old tracks before deleting it [s]
      delay_tolerance: 0.05     # delay tolerance for track timeout [s]
      publish_rate: 15.0        # Rate at which to publish and check whether a track is old [Hz]
      field_of_view: 2.0944     # Robot camera field of view in the real world [radians]
      agent_radius: 0.2         # Radius of agent for evaluator node 